packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2091,19 +2135,19 @@ describe('plugin-meetings', () => {
 
         describe('updateMedia()', () => {
 
-          const addMedia = async (enableMedia, track) => {
-            await meeting.addMedia({audioEnabled: enableMedia, localTracks: {microphone: track}});
+          const addMedia = async (enableMedia, stream) => {
+            await meeting.addMedia({audioEnabled: enableMedia, localStreams: {microphone: stream}});
             await simulateRoapOffer();
 
             resetHistory();
           }
 
           const checkAudioEnabled = (expectedTrack, expectedDirection) => {
             if (isMultistream) {
-              assert.calledOnceWithExactly(fakeMultistreamRoapMediaConnection.enableMultistreamAudio, expectedDirection !== 'inactive');
+              assert.calledOnceWithExactly(meeting.sendSlotManager.setActive, MediaType.AudioMain, expectedDirection !== 'inactive');
```

Comment:
same as above, would be better to check setActive being called on the right sendSlot

Reply:
Commented above